<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1021</width>
     <height>571</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lab_time">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_tem">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tem_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--.--</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hum">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_sum">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>光强</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nh3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>氨气</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hum_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>--.--</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_sun_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>--.--</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nh3_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>--.--</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic_tem">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>25</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic_hum">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic_sun">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic_nh3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>消息窗口</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_echo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="discon_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>命令3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_cmd_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>命令2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>title</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>691</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QPushButton" name="send_cmd_btn_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QComboBox" name="send_time_for_histoty">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查找历史记录</string>
     </property>
    </widget>
    <widget class="QChartView" name="chartView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>661</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
